docs/tutorials/localization.rst

```diff
@@ -0,0 +1,74 @@
+Localization with points & lines
+=================================
+
+Currently, runner scripts are provided to run visual localization integrating line along with point features on the following Datasets: 
+
+* `7Scenes Dataset <https://www.microsoft.com/en-us/research/project/rgb-d-dataset-7-scenes/>`_
+* `Cambridge Landmarks Dataset <https://www.repository.cam.ac.uk/handle/1810/251342>`_
```

Comment:
The link to the Cambridge dataset redirects me to a blank page. Maybe we can instead put the link to the original paper: https://arxiv.org/pdf/1505.07427v4.pdf